scripts/pre_push_hook.py

```diff
@@ -94,6 +103,33 @@ def _start_subprocess_for_result(cmd):
     return out, err
 
 
+def _get_remote_name():
+    """Get the remote name of the local repository.
+    Returns:
+        str. The remote name of the local repository.
+    """
+    remote_name = ''
+    remote_num = 0
+    # Get the current repository.
+    repo = git.Repo(os.getcwd())
+    for remote in repo.remotes:
+        remote_url = list(remote.urls)[0]
+        if remote_url.endswith('oppia/oppia.git'):
+            remote_num += 1
+            remote_name = remote.name
+
+    if remote_num == 0:
```

Comment:
You can remove the equality check, 0 will evaluate to false.

Reply:
I don't understand this -- We are checking if there is an upstream set or not. Can you please come again? Thanks

Reply:
My bad, make it `if not remote_num`.